File: src/test/java/dev/retrotv/random/generator/ByteGeneratorTest.java
package dev.retrotv.random.generator;

import org.junit.jupiter.api.DisplayName;
import org.junit.jupiter.api.Test;

import java.security.SecureRandom;

import static org.junit.jupiter.api.Assertions.assertNotNull;

class ByteGeneratorTest {

    @Test
    @DisplayName("ByteGenerator 테스트")
    void test_byteGenerator() {
        ByteGenerator byteGenerator = new ByteGenerator(new SecureRandom());
        assertNotNull(byteGenerator.generate());
    }
}

File: src/test/kotlin/dev/retrotv/random/generator/ByteGeneratorTest.kt
package dev.retrotv.random.generator

import java.security.SecureRandom
import kotlin.test.Test
import kotlin.test.assertNotNull

class ByteGeneratorTest {

    @Test
    fun test_ByteGenerator() {
        val byteGenerator = ByteGenerator(SecureRandom())
        assertNotNull(byteGenerator.generate())
    }
}